src/server/csp.ts

```diff
@@ -65,24 +65,30 @@ const assetHashes = (assets: string[]): string =>
 //     child-src https: blob:
 // `.trim()
 
-const buildCsp = ({ styles, scripts }: Assets, twitter: boolean): string => `
+const styleSrc = (styles: string[], twitter: boolean, hasInlineStyles: boolean) => {
+    return hasInlineStyles
+        ? `https: 'unsafe-inline';`
+        : `'self' ${assetHashes(styles)} https://interactive.guim.co.uk ${twitter ? 'https://platform.twitter.com' : ''};`
+}
```

Comment:
Hmm, this narrows the issue to a subset of articles, but it still drops much of the CSP security for any articles with these embeds right? 🤔

Also, if we were to go with the `unsafe-inline` approach, wouldn't we still want to narrow the external domains? By that I mean we'd want to allow specific domains rather than whitelisting all `https:`?